src/main/java/frc/robot/Robot.java

```diff
@@ -4,9 +4,15 @@
 
 package frc.robot;
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.commands.drivingCommands.ArcadeDriveCommand;
+import frc.robot.subsystems.drive.Drive;
+import frc.robot.subsystems.drive.Gyroscope;
+import frc.robot.operatorInputs.Controls;
```

Comment:
Make sure all these imports in green are being used when you open it up in vscode. They'll be grayed out if they are not being used.